client/js/options.js

```diff
@@ -1,6 +1,7 @@
 "use strict";
 
 const $ = require("jquery");
+require("jquery-textcomplete");
```

Comment:
Very odd that you've had build issues with this. But this is an issue unrelated to this PR so please remove it.

Reply:
Removed